pypet2bids/README.md

```diff
@@ -0,0 +1,132 @@
+
+Simply download the repository - follow the specific Matlab or Python explanations. Matlab and Python codes provide the same functionalities.
+
+### matlab
+
+[![asciicast](https://asciinema.org/a/RPxiHW6afISPmWYFBOGKWNem1.svg)](https://asciinema.org/a/RPxiHW6afISPmWYFBOGKWNem1)
+
+1) remember to set the path to the PET2BIDS/matlab folder, you will find the source code to use here.
+2) if converting DICOM files, make sure you have dcm2niix (for windows users, edit dcm2niix4pet.m to set the right paths to the .exe)
+3) start using the code! more info [here](https://github.com/openneuropet/PET2BIDS/tree/main/matlab#readme)
+
+### pypet2bids
+
+Use pip to install this library directly from PyPI:
+
+[![asciicast](https://asciinema.org/a/TZJg5BglDMFM2fEEX9dSpnJEy.svg)](https://asciinema.org/a/TZJg5BglDMFM2fEEX9dSpnJEy)
+
+If you wish to install directly from this repository see the instructions below to either build
+a packaged version of `pypet2bids` or how to run the code from source.
+
+<details>
+<summary>Build Package Locally and Install with PIP</summary> 
+
+We use [poetry](https://python-poetry.org/) to build this package, no other build methods are supported, 
+further we encourage the use of [GNU make](https://www.gnu.org/software/make/) and a bash-like shell to simplify the 
+build process.
+
+After installing poetry, you can build and install this package to your local version of Python with the following 
+commands (keep in mind the commands below are executed in a bash-like shell):
+
+```bash
+cd PET2BIDS
+cp -R metadata/ pypet2bids/pypet2bids/metadata
+cp pypet2bids/pyproject.toml pypet2bids/pypet2bids/pyproject.toml
+cd pypet2bids && poetry lock && poetry build
+pip install dist/pypet2bids-X.X.X-py3-none-any.whl
+```
+
+Why is all the above required? Well, because this is a monorepo and we just have to work around that sometimes.
+
+
+[!NOTE]
+Make and the additional scripts contained in the `scripts/` directory are for the convenience of 
+non-windows users.
+
+If you have GNU make installed and are using a bash or something bash-like in you your terminal of choice, run the 
+following:
+
+```bash
+cd PET2BIDS
+make installpoetry buildpackage installpackage
+```
+
+</details>
+
+<details> 
+<summary>Run Directly From Source</summary>
+
+Lastly, if one wishes run pypet2bids directly from the source code in this repository or to help contribute to the python portion of this project or any of the documentation they can do so via the following options:
+
+```bash
+cd PET2BIDS/pypet2bids
+poetry install
+```
+
+Or they can install the dependencies only using pip:
+
+```bash
+cd PET2BIDS/pypet2bids
+pip install .
+```
+
+After either poetry or pip installation of dependencies modules can be executed as follows:
+
+```bash
+cd PET2BIDS/pypet2bids
+python dcm2niix4pet.py --help
+```
+
+</details>
+
+**Note:**
+*We recommend using dcm2niix v1.0.20220720 or newer; we rely on metadata included in these later releases. It's best to 
```

Comment:
Just for clarification: do you need v > 1.0.20220720 because you need additional metadata, or pet2bids will run with older version, but will miss some meta?

Reply:
The latter, I'd have to do some digging to remember exactly what fields are missing in the earlier versions. It's recommended as an older version wont "break" the software, but it will require extra effort on the users part to make it bids compliant as they'll have to supply the missing metadata on their own.